templates/meta_woz/dialogues/templates.yaml

```diff
@@ -4,7 +4,8 @@ templates:
   4b64c6e9-0aa0-431f-85b5-8367daa4773a: !Template
     id: 4b64c6e9-0aa0-431f-85b5-8367daa4773a
     jinja: "What does this conversation between a chatbot and a client talk about\
-      \ ? \n{{turns | join(',  ')}}\n|||\n{{domain.replace('_', ' ') | lower }}"
+      \ ? \n{{turns |  join(\"\\n- \")}}\n|||\n{{domain.replace('_', ' ') | lower\
```

Comment:
When you use join like this, the first line will not get a - before it, but all the other ones will. Can you just do the human/bot labels like you did in the other tasks?